DependencyInjection/Configuration.php

```diff
@@ -388,6 +403,14 @@ private function addExceptionSection(ArrayNodeDefinition $rootNode)
                             ->useAttributeAsKey('name')
                             ->prototype('boolean')->end()
                         ->end()
+                        ->booleanNode('show_exception')
+                            ->treatNullLike($this->debug)
+                            ->defaultValue($this->debug)
+                        ->end()
+                        ->booleanNode('debug_message')
```

Comment:
One _show_exception_ forces to add exception to error response
Second _debug_message_ forces to use exception message instead of standard response status text
Maybe second config param should renamed to _show_exception_message_ to be more clear